cookbook/rule/general_information_on_rule_format.rst

```diff
@@ -0,0 +1,562 @@
+General information on rule format
+==================================
+
+Quick Overview
+--------------
+
+**This cookbook is about a feature only provided in the Enterprise Edition.**
+
+Enrichment rules enable calculating the value of an attribute according specific conditions. Those rules are regularly
+applied, as a user who would regularly edit product.
+
+File Structure
+--------------
+
+Enrichment rules are defined in YAML. The file extension has to be ".yml". Indentation is mandatory within the
+file and has to follow the YAML format strictly. To use a rule it has to be imported in the PIM.
+
+This file starts with "rules" root element, which contains the list of enrichment rules. This document is about this
+list. Each rule is referred to by a code and can contain a list of conditions or actions. Actions are executed if all
+conditions are verified or if no action is defined.
+
+.. code-block:: yaml
+
+    # Example of a file with 2 rules
+    rules:
+        camera_set_canon_brand:
+            priority: 0
+            conditions:
+                - field: family.code
+                  operator: IN
+                  value:
+                    - camcorders
+                - field: name
+                  operator: CONTAINS
+                  value: Canon
+            actions:
+                - type: set_value
+                  field: camera_brand
+                  value: canon_brand
+        camera_copy_name_to_model:
+            priority: 0
+            conditions:
+                - field: family.code
+                  operator: IN
+                  value:
+                    - camcorders
+            actions:
+                - type: copy_value
+                  from_field: name
+                  to_field: camera_model_name
+
+Indentation is mandatory within the file and must strictly be identical as the one shown in the example.
+
+.. tip::
+
+    For a rule without conditions, set the following condition:
+
+    .. code-block:: yaml
+
+        conditions:[]
+        actions:
+          - type:
+
+Enrichment Rule Structure
+-------------------------
+
+Structure’s elements which define a rule are :
+ - rule code (dynamic)
+ - priority*
+ - conditions
+ - field
+ - locale*
+ - scope*
+ - operator
+ - value
+ - actions
+ - type
+
+An enrichment rule is structured as follows :
+
+.. code-block:: yaml
+
+    [free rule code]:
+        priority​*:
+        conditions:
+            - field:
+              locale​*:
+              scope​*:
+              operator:
+              value:
+        actions:
+            - type:
+              [Diverse elements according to the action]
+
+Elements with * are optionals.
+
+**Dashes** - ​before element field and after each element contained in value part are mandatory.
+
+**Colon** : ​mandatory after each structure element.
+
```

Comment:
for more details see the yaml specification (add a link to the yaml documentation)